cleansweep/plugins/committees/templates/edit_committee_structure.html

```diff
@@ -6,9 +6,12 @@
   <ul class="breadcrumb breadcrumb-collapse">
     <li><a href="{{url_for('admin')}}">Admin Center</a></li>
     <li><a href="{{url_for('.committee_structures')}}">Committee Structures</a></li>
+    <li><a href="{{url_for('.committees_by_level', level=committee_type.place_type.short_name)}}"
+        >{{ committee_type.place_type.name }}</a></li>
+    <li><a href="{{ committee_type.url_for('.view_committee_structure') }}">{{ committee_type.name }}</a></li>
     <li></li>
   </ul>
-  <h2>Edit <a href="{{committee_type.url_for('.view_committee_structure')}}">{{committee_type.name}}</a></h2>
+  <h2>Edit</h2>
```

Comment:
Could you please keep the earlier version? 

The heading is what takes people attention. They should know by what they are editing by looking at it. Since we are already linking it in h2, there is no need for adding it to the breadcrumbs.